internal/config/reloader/remote.go

```diff
@@ -49,7 +49,7 @@ type RemoteConfigReloader struct {
 // Do starts the remote config reloader.
 func (u *RemoteConfigReloader) Do(ctx context.Context) error {
 	u.log.Info("Starting...")
-
+	u.interval = time.Second * 10
```

Comment:
Is this override for testing purposes?

Reply:
right, I reverted it. Botkube fails with default settings since it does not have this interval value.